### ISSUE TITLE
Question about DRF client id and secret?

### ISSUE THREAD

**Issue by retzero** [Ronakchitlangya1997/Social-Authentication-Email-using-Django-and-React-JS#1]
Hello.
Thank you for this great post!

Where can I find this two values to set in `login.js` file?
`DRF client id` and `DRF client secret`.

Thanks.

**Comment by mahiuddin-dev** [comment]
> Hello. Thank you for this great post!
> 
> Where can I find this two values to set in `login.js` file? `DRF client id` and `DRF client secret`.
> 
> Thanks.

Yes same question here also what is **DRF client id** and **DRF client secret**
